cmd/listagreements/listagreements.go

```diff
@@ -82,6 +82,9 @@ func (c *listAgreementsCommand) Run(ctx *cmd.Context) error {
 
 	agreements, err := apiClient.GetUsersAgreements()
 	if err != nil {
+		if errors.IsNotFound(errors.Cause(err)) {
+			return errors.Errorf("no agreement found")
```

Comment:
I'd say don't return an error - return that we know exactly what's happening, but that the user has no agreements